lib/src/screens/dashboard/pages/balance_page.dart

```diff
@@ -506,6 +515,49 @@ class BalanceRowWidget extends StatelessWidget {
                   ),
                 ],
               ),
+            if (hasAdditionalBalance)
```

Comment:
wouldn't this also affect Monero wallet?

Reply:
Yes it would, added another check to make sure this impacts only BTC and LTC